sklearn/manifold/_t_sne.py

```diff
@@ -681,12 +688,30 @@ def __init__(self, n_components=2, *, perplexity=30.0,
     def _fit(self, X, skip_num_points=0):
         """Private function to fit the model using X as training data."""
 
+        if self.init == 'warn':
+            # See issue #18018
+            warnings.warn("The default initialization will change from "
+                          "random to PCA in 1.01.", FutureWarning)
```

Comment:
We will name versions 1.0, 1.1, 1.2, ... We also wait two releases to change behaviors.
```suggestion
            warnings.warn("The default initialization in TSNE will change "
                          "from 'random' to 'pca' in 1.2.", FutureWarning)
```